automatic/1password/tools/chocolateyInstall.ps1

```diff
@@ -37,3 +37,6 @@ Start-Job -ScriptBlock { param($cache_dir)
   }
 } -ArgumentList ($cache_dir)
 Install-ChocolateyPackage @packageArgs
+if ($env:ChocolateyPackageName -ne "1password4") {
```

Comment:
While it is true that the `1password4` and `1password` packages is handled in this same location in the repository here, this was originally due to them both being the same just having the exact same logic back then except for the silent arguments used (which are inserted during the update).

The end results when pushed to the community repository should only include the logic that is necessary for the package that was pushed.

Since these two packages now differ, changes would need to be made to either

1. Remove the streams, and make them two separate packages in the repository altogether (common logic could still be shared between helper scripts).
2. Have two install/uninstall scripts outside of the tools directory, and during the updated copy the appropriate one to the tools directory so it is used.

So while I understand why you added the if statement here, it would still not be right in terms of the end result being uploaded.

Reply:
This is still an issue. I see the issue with this is down to the password4 thing. I've ran line 4. I'm not sure why the two versions are running of the same install file but does anyone know enough about this and choco's systems to make the correction @AdmiringWorm surgested?